specification/cognitiveservices/resource-manager/Microsoft.CognitiveServices/preview/2023-10-01-preview/cognitiveservices.json

```diff
@@ -5631,25 +5631,16 @@
       "description": "Azure OpenAI Content Filter.",
       "type": "object",
       "properties": {
-        "policyName": {
+        "name": {
           "type": "string",
           "description": "Name of Content Filter."
         },
-        "description": {
-          "type": "string",
-          "description": "Description of Content Filter."
+        "isMultiLevelFilter": {
```

Comment:
>isMultiLevelFilter [](http://example.com/codeflow?start=9&length=18)

From ARM perspective, this is moving the wrong direction. Why do you need to change from enum to boolean? Enums are generally better properties and highly recommended over booleans in ARM APIs.

Reply:
Yes, why change this